src/inference/emcee.jl

```diff
@@ -0,0 +1,243 @@
+###
+### Sampler states
+###
+
+struct Emcee{space, E<:AMH.Ensemble} <: InferenceAlgorithm 
+    ensemble::E
+end
+
+function Emcee(n_walkers::Int, stretch_length=2.0)
+    # Note that the proposal distribution here is just a Normal(0,1)
+    # because we do not need AdvancedMH to know the proposal for
+    # ensemble sampling.
+    prop = AMH.StretchProposal(Normal(0, 1), stretch_length)
```

Comment:
I guess type restrictions in AdvancedMH don't allow to use
```suggestion
    prop = AMH.StretchProposal(nothing, stretch_length)
```
to make it clear that `Normal(0, 1)` should never be used by any method?

Reply:
Actually, I don't think there's any such restriction. I think your proposal should work.